src/pages/Home.tsx

```diff
@@ -112,17 +111,23 @@ const Home: FC = () => {
 
     setCanShowAvatar(canShowAvatarValue);
   };
+  const urlParams = new URLSearchParams(location.search);
   const [canShowAvatar, setCanShowAvatar] = useState<boolean>();
-
+  const [currentHeader, setCurrentHeader] = useState(() => {
+    const currPage = urlParams.get("tab");
+    if (currPage === HOMEHEADERLIST.HOME) {
```

Comment:
don't have to check for only 'HOME' header 
just check if currPage is of HOMEHEADERLIST enum 
if that is HOMEHEADERLIST enum set it to  useState